fieldMapping.json

```diff
@@ -2,39 +2,39 @@
   "Version": "v1",
   "MappingFields": [
     {
```

Comment:
id field is missing for aRef on accountdata
Recommend adding this to de fieldMapping on github

Reply:
We discussed this and as it's a bigger change than just the mapping, we decided to leave this out